commons/com.b2international.index.lucene/src/com/b2international/index/json/JsonDocumentSearcher.java

```diff
@@ -111,7 +109,7 @@ public void close() throws Exception {
 			return searchDefault(query);
 		}
 		
-		if (!SortBy.NONE.equals(query.getSortBy())) {
+		if (!SortBy.DOC.equals(query.getSortBy())) {
```

Comment:
Remove this check after merging #139 to develop and synchronizing this PR with develop.